core-java-modules/core-java-jvm/pom.xml

```diff
@@ -64,6 +57,24 @@
         </dependency>
     </dependencies>
 
+<!--    <build>-->
```

Comment:
can we remove this entirely if not needed? let's not have commented code

Reply:
Done